views/index.html

```diff
@@ -1,29 +1,190 @@
+          <h5 class="modal-title" id="exampleModalLabel">Add new gym/review</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+
+
+
+
+          <form class="form-horizontal">
+            <fieldset>
+
+              <!-- Text input-->
+              <div class="form-group">
+                <label class="col-md-4 control-label" for="name">Gym Name</label>
+                <div class="col-md-4">
+                  <input id="name" name="gymName" type="text" placeholder="" class="form-control input-md" required="">
+
+                </div>
+              </div>
+
+              <!-- Text input-->
+              <div class="form-group">
+                <label class="col-md-4 control-label" for="artistName">Location</label>
+                <div class="col-md-4">
+                  <input id="textinput" name="gymLocation" type="text" placeholder="" class="form-control input-md">
+
+                </div>
+              </div>
+
+              <!-- Text input-->
+              <div class="form-group">
+                <label class="col-md-4 control-label" for="releaseDate">Image url</label>
+                <div class="col-md-4">
+                  <input id="releaseDate" name="image" type="text" placeholder="" class="form-control input-md">
+
+                </div>
+              </div>
+
+              <!-- Textarea -->
+              <div class="form-group">
+                <label class="col-md-4 control-label" for="reviews">Review:</label>
+                <div class="col-md-4">
+                  <textarea class="form-control" id="reviewTxt" name="reviews">This place sucked</textarea>
+                </div>
+              </div>
+
+
+
+            </fieldset>
+         
+
+
+
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+          <button type="submit" class="btn btn-primary saving">Post review</button>
         </div>
+         </form>
       </div>
     </div>
-  </body>
+  </div>
+<!-- end of modal -->
+
+
+
+
+
+<!-- jumbotron header -->
+  <div class="container">
+    <div class="jumbotron">
+      <h1 class="display-3">Review a Jiu-Jitsu Academy</h1>
+      <p>Let everyone know if it was legit or not!</p>
+      <p class="lead">It is simple, just click the button below, fill out the form and click submit.</p>
+      <hr class="my-4">
+      <p>  Please keep reviews clean!</p>
+      <p class="lead">
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+         Add gym
+       </button>
+       <!-- <a class="btn btn-primary btn-lg" href="#" role="button">Add gym</a> -->
+     </p>
+   </div>
+ </div>
+
+
+
+ 
+
+<!-- gym data display div -->
+
+<div id="gymDisplay"></div>
+<!-- <div class="container">
```

Comment:
this much commented out code is usually a bad sign--better to take it out of this file entirely.